tests/test_integration/test_dace_parsing.py

```diff
@@ -41,6 +41,10 @@ def dace_env():
     with dace.config.temporary_config():
         # Setting max_concurrent_streams to -1 configures dace to only use the default stream.
         dace.config.Config.set("compiler", "cuda", "max_concurrent_streams", value=-1)
+        dace.config.Config.set(
+            "compiler", "cuda", "args", value="-Xcompiler -fPIC --default-stream legacy"
+        )
```

Comment:
Not sure how much this matters, but `-fPIC` is not the flag name on all host compilers (e.g. Microsoft and IBM compilers use different names). Is that needed, or only the `--default-stream` option?

Reply:
I put back the -fPIC since the error message before asked for it :sweat_smile: 
But the original default flags also had the -fPIC so this should be harmless.

Reply:
Ah, good to know!